server/views/pages/bookings/visit.njk

```diff
@@ -71,6 +72,16 @@
       <p class="govuk-!-margin-bottom-1" data-test="main-contact-name">{{ visit.visitContact.name }}</p>
       <p data-test="main-contact-number">{{ visit.visitContact.telephone | d("No phone number provided", true) }}</p>
 
+      {% if showCancel %}
+        {{ govukButton({
+          text: "Cancel booking",
+          classes: "govuk-!-margin-top-5 govuk-button--secondary",
+          href: paths.BOOKINGS.CANCEL.CANCEL_VISIT + '/' + visit.visitDisplayId,
```

Comment:
This is why I suggested removing the upper `{ CANCEL: ...` bit, `CANCEL.CANCEL_VISIT` looks strange, either `CANCEL.VISIT` or `CANCEL_VISIT`, with the latter matching what everything else does.